src/main/java/org/isf/lab/rest/LaboratoryController.java

```diff
@@ -294,7 +297,7 @@ public ResponseEntity<List<LabWithRowsDTO>> getLaboratory(@PathVariable Integer
 			throw new OHAPIException(new OHExceptionMessage(null, "Patient not found!", OHSeverityLevel.ERROR));
 		}
 
-		List<Laboratory> labList = laboratoryManager.getLaboratory(patient);
+		List<Laboratory> labList = laboratoryManager.getLaboratory(patient).stream().filter(e -> !e.getStatus().equals(LaboratorySTATUS.DRAFT.toString())).collect(Collectors.toList());
 		if (labList == null || labList.isEmpty()) {
 			return ResponseEntity.status(HttpStatus.NO_CONTENT).body(null);
 		} else {
```

Comment:
There really isn't a need to have an `else` when the previous statement is `return`.